modules/services/cloud-logs/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "bucket_name" {
+  description = "(Required) The name of your s3 bucket associated with your Clloudtrail trail"
+  type        = string
+}
+
+variable "external_id" {
+  type        = string
+  description = "(Required) Random string generated unique to a customer"
+}
+
+variable "region" {
+  description = "(Optional) Default region for resource creation"
+  type        = string
+  default     = "eu-central-1"
```

Comment:
This default value was inherited from the previous `trust-relationship` config and related to organization mode. Does this still hold, considering that we won't even support organizational accounts right from the start?